src/query/service/src/table_functions/others/task_dependents_enable.rs

```diff
@@ -0,0 +1,199 @@
+// Copyright 2021 Datafuse Labs
```

Comment:
It's better to spill out all the cloud related table functions into a new folder like `cloud`